MAINTAINERS.yml

```diff
@@ -2859,17 +2859,21 @@ RISCV arch:
     - carlocaione
     - npitre
     - ycsin
+    - pfarwsi
+    - kanakshilledar
```

Comment:
Are you already planning to help with all RISC-V related work in the project? I.e. review the incoming PRs etc?
Given this looks like your first contribution, it might be a bit premature IMO
Not sure who's the other person but I'm assuming (perhaps wrongly) the same might be true

Reply:
Yes we have already planned for the RISC-V related work going forward which would be adding more features for this board, looking into issues and also review the incoming PR's. Since RISC-V area is quite new for everyone and needs more and more support, we thought of contributing this small piece of work to the community :). There would be more and more work coming forward in this area. This is a joint activity between @pfarwsi and @kanakshilledar.

Reply:
I have to agree with @kartben on this one. I've noticed that the usual path people take to be responsible for an area of Zephyr is as follows:
1. contribute & in general - interact with the community (you're here)
2. start reviewing other people's code
3. become a collaborator
4. become a maintainer

We're of course more than happy to have you and your contributions here. That being said, being responsible for the RISC-V architecture as a whole means you need to have the broader picture, be aware of all the different conventions the project conforms to and so on. And I'm just not sure if that's true for you, having only one PR under your belt.

IMO a smaller area would be better for a new member of the community, i.e. a StarFive-specific one (e.g. see the ITE platforms area) which @ycsin proposed here: https://github.com/zephyrproject-rtos/zephyr/pull/69814/files#r1514001131.

I'm unresolving this thread for now, as this IMO needs more discussion.